src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -21,7 +21,46 @@ public class PackageManagerSearchElementViewModel : BrowserItemViewModel, IEquat
         public ICommand VisitRepositoryCommand { get; set; }
         public ICommand DownloadLatestToCustomPathCommand { get; set; }
 
-        public new PackageManagerSearchElement Model { get; internal set; }
```

Comment:
oh I see, you removed it and the new keyword...what was it hiding?

Reply:
Yup, exactly! It was hiding it.